opencti-platform/opencti-graphql/src/domain/stixCyberObservable.js

```diff
@@ -143,13 +143,13 @@ const createIndicatorFromObservable = async (context, user, input, observable) =
         externalReferences: input.externalReferences,
         update: true,
       };
-      await addIndicator(context, user, indicatorToCreate);
-    } else {
-      logApp.warn('Cannot create indicator - cant generate pattern', { key, value });
+      return addIndicator(context, user, indicatorToCreate);
```

Comment:
what happens if addIndicator throws an error here ? I guess it won't be catched here, and it will throw an error where it is called, but then since it's called at the end of addStixCyberObservable, it might fail the observable creation even though the observable has been created.